rllib/models/experimental/base.py

```diff
@@ -0,0 +1,96 @@
+from dataclasses import dataclass
+import abc
+
+from ray.rllib.models.specs.checker import (
+    check_input_specs,
+    check_output_specs,
+)
+from ray.rllib.models.specs.specs_dict import SpecDict
+from ray.rllib.models.temp_spec_classes import TensorDict
+from ray.rllib.utils.annotations import ExperimentalAPI
+
+ForwardOutputType = TensorDict
+
+
+@ExperimentalAPI
+@dataclass
+class ModelConfig(abc.ABC):
```

Comment:
As discussed offline yesterday, we'll keep the build method because it abstracts the class to be built. Any model_config infering code will simply return a config that can be built. We'd otherwise have to return a class (which we don't want because it's not framework agnostic) to resolve this issue.